google-cloud-pubsub/OVERVIEW.md

```diff
@@ -316,6 +316,89 @@ received_messages = sub.pull
 sub.modify_ack_deadline 120, received_messages
 ```
 
+## Using Ordering Keys
+
+Google Cloud Pub/Sub ordering keys provide the ability to ensure related
+messages are sent to subscribers in the order in which they were published.
+Messages can be tagged with an ordering key, a string that identifies related
+messages for which publish order should be respected. The service guarantees
+that, for a given ordering key and publisher, messages are sent to subscribers
+in the order in which they were published. Ordering does not require sacrificing
+high throughput or scalability, as the service automatically distributes
+messages for different ordering keys across subscribers.
+
+Note: At the time of this release, ordering keys are not yet publicly enabled
+and requires special project enablements. To participate contact the Pub/Sub
```

Comment:
How can an external customer "contact the pub/sub team"? If this is limited to certain partners, we should say so and tell them to contact their account manager. Otherwise, if it should be available for any customer, we should provide a link or something, because there isn't otherwise an obvious way for an arbitrary external customer to contact an internal team.

Reply:
@kamalaboulhosn

Reply:
In other languages we did not specify to "contact the Pub/Sub team." There isn't a public link that can be used for this. I'd rather keep it ambiguous. Can we just go with "Ordering is not yet fully supported and requires special project enablements?" Generally we've been reaching out to customers via CEs or via a list we have of people who have expressed interest over time.

Reply:
Updated and removed the sentence about contacting the team. PTAL.